### ISSUE TITLE
Bridge wrapper shoud not push bound value to kendo widget if it already has the correct value

### ISSUE THREAD

**Issue by sylvain-hamel** [aurelia-ui-toolkits/aurelia-kendoui-bridge#642]
In widgets that have a value (dropdown, textbox, etc.) when the value is changed in the kendo widget the bridge wrapper reacts accordingly. In the same manner, when the bridge wrapper value is changed, the bridge wrapper pushes the value to the kendo widget. In reaction, the kendo widget "does its thing". In the case of a dropdown, it may read the data source again and reapply filters. With the current bridge implementation, when the bridge wrapper receives a value it does not check if the value is already the right one on the kendo widget and assigns it anyways. The consequence is that the dropdown will "do its things" uselessly. That triggers a lot of code that does not need to be executed (and that causes an issue in my project with reevaluating the server-side datasource filters).

I'd like `widgetBase` to only push the change to the kendo widget if the value is different. The [actual code] (https://github.com/aurelia-ui-toolkits/aurelia-kendoui-bridge/blob/master/src/common/widget-base.js#L272) is:

```
  handlePropertyChanged(widget, property, newValue, oldValue) {
    let binding = this.bindingsToKendo.find(i => i.propertyName === property);
    if (binding) {
      widget[binding.functionName](newValue);
    }
  }
```

And I'd like to have it changed to something like this:

```
  handlePropertyChanged(widget, property, newValue, oldValue) {
    let binding = this.bindingsToKendo.find(i => i.propertyName === property);
    if (binding) {
        if (widget && widget[binding.functionName]() !== newValue) {
          widget[binding.functionName](newValue);
        }
    }
  }
```

**Comment by sylvain-hamel** [comment]
Thanks!

**Comment by JeroenVinke** [comment]
Will let you know when the release is published

**Comment by JeroenVinke** [comment]
Released as 1.0.0-beta.1.0.16
